src/AppBundle/Resources/views/themes/app/pagelayout.html.twig

```diff
@@ -32,7 +32,7 @@
 
 {% include '@NetgenSite/parts/facebook_api.html.twig' %}
 
-<div id="page"{% if page_css_class|default is not empty %} class="{{ page_css_class }}"{% endif %}>
+<div id="page"{% if page_css_class|default is not empty %} class="{{ page_css_class }}"{% endif %}{% if location is defined and location is not empty %} data-path='["{{ location.path|join('","') }}"]'{% endif %}>
```

Comment:
@emodric I have completely remove the pagelayout_variables block, I am only doing the check for the location object here (exists and not empty) so it doesn't crash on module pages like search or tag view. Raw was removed as well as the slicing of the system root node.